code/modules/jobs/job_types/bos.dm

```diff
@@ -470,7 +425,7 @@ Star Paladin
 /datum/outfit/loadout/spaladinc
 	name = "Senior Frontline Paladin"
 	backpack_contents = list(
-		/obj/item/gun/energy/laser/aer12 = 1,
+		/obj/item/gun/energy/laser/aer9 = 1,
 		/obj/item/gun/energy/laser/pistol=1,
```

Comment:
```suggestion
		/obj/item/gun/energy/laser/pistol=1,
```
Suggest replacing this with an upgraded wattz 1k

Reply:
Wattz is still the most powerful laser rifle hands down. Would rather not.
If it doesn't work out, we'll up it back to an AER12 or, perhaps, yes, a Wattz.

Reply:
> Wattz is still the most powerful laser rifle hands down. Would rather not. If it doesn't work out, we'll up it back to an AER12 or, perhaps, yes, a Wattz.

Watts 1k is the pistol version